sources/packages/api/src/route-controllers/institution/institution.aest.controller.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  Body,
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  Patch,
+  HttpStatus,
+} from "@nestjs/common";
+import { InstitutionService } from "../../services";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import {
+  InstitutionProfileDto,
+  InstitutionReadOnlyDto,
+} from "./models/institution.dto";
+import BaseController from "../BaseController";
+import { ClientTypeBaseRoute } from "../../types";
+import { InstitutionControllerService } from "./institution.controller.service";
+import { ApiTags, ApiOperation, ApiResponse } from "@nestjs/swagger";
+/**
+ * Institution controller for AEST Client.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("institution")
+@ApiTags("institution")
+export class InstitutionAESTController extends BaseController {
+  constructor(
+    private readonly institutionService: InstitutionService,
+    private readonly institutionControllerService: InstitutionControllerService,
+  ) {
+    super();
+  }
+
+  /**
+   * Get institution details of given institution.
+   * @param institutionId
+   * @returns InstitutionReadOnlyDto
+   */
+  @Get("/:institutionId")
+  @ApiOperation({
```

Comment:
We no longer need _@ApiOperation_ since the comments can be automatically generated.

Reply:
I see that comments weren't automatically generated for an API unless we use this decorator. Am I missing something? 
![image](https://user-images.githubusercontent.com/54600590/158503978-240de3b9-42bb-4ccb-863b-c5019e26702a.png)